requirements/requirements.sam2.txt

```diff
@@ -0,0 +1,4 @@
+rasterio~=1.3
+torch
```

Comment:
Just a note - should the version of torch be pulled to match CUDA environment?

Reply:
That happens in the dockerfile

Reply:
Is this pip installable also though?